static/src/javascripts/boot.js

```diff
@@ -1,5 +1,12 @@
 // @flow
 
+// es7 polyfills not provided by pollyfill.io
+import 'core-js/modules/es7.object.values';
```

Comment:
This should hopefully be in the next release -- https://github.com/Financial-Times/polyfill-service/pull/1082